vendor/engines/badges_engine/spec/models/badges_engine/level_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+module BadgesEngine
+  describe Level do
+    let(:level) { build(:level) }
+
+    it { expect(level).to be_valid }
+
+    context 'validations' do
+      it { expect(level).to validate_numericality_of(:tier).only_integer }
+    end
+
+    context 'relationships' do
+      it { expect(level).to belong_to(:badge) }
+    end
+
+    context 'after initialize' do
+      it "generates a 1 tier level on initialize" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.